Partial import of upstream git-commit to fix build in c++11 mode

From 92ec47e15b62ef12e15e6ce5550abe2620bc3167 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 2 Jul 2014 14:17:19 +0000
Subject: [PATCH] Fix build when using C++11

--- modules/access/live555.cpp.orig	2015-10-21 17:48:45 UTC
+++ modules/access/live555.cpp
@@ -307,7 +307,7 @@ static int  Open ( vlc_object_t *p_this
     p_demux->p_sys     = p_sys = (demux_sys_t*)calloc( 1, sizeof( demux_sys_t ) );
     if( !p_sys ) return VLC_ENOMEM;

-    msg_Dbg( p_demux, "version "LIVEMEDIA_LIBRARY_VERSION_STRING );
+    msg_Dbg( p_demux, "version " LIVEMEDIA_LIBRARY_VERSION_STRING );

     TAB_INIT( p_sys->i_track, p_sys->track );
     p_sys->f_npt = 0.;
@@ -593,7 +593,7 @@ createnew:

     p_sys->rtsp = new RTSPClientVlc( *p_sys->env, psz_url,
                                      var_InheritInteger( p_demux, "verbose" ) > 1 ? 1 : 0,
-                                     "LibVLC/"VERSION, i_http_port, p_sys );
+                                     "LibVLC/" VERSION, i_http_port, p_sys );
     if( !p_sys->rtsp )
     {
         msg_Err( p_demux, "RTSPClient::createNew failed (%s)",
 modules/access/live555.cpp                   |  4 ++--
 modules/demux/mp4/libmp4.h                   |  2 +-
 modules/demux/vobsub.h                       |  8 +++----
 10 files changed, 40 insertions(+), 40 deletions(-)

--- modules/demux/mp4/libmp4.h.orig	2015-02-02 19:42:29 UTC
+++ modules/demux/mp4/libmp4.h
@@ -1360,7 +1360,7 @@ static inline size_t mp4_box_headersize(
     if( i_actually_read < 0 || (int64_t)i_actually_read < i_read )\
     { \
         msg_Warn( p_stream, "MP4_READBOX_ENTER: I got %i bytes, "\
-        "but I requested %"PRId64"", i_actually_read, i_read );\
+        "but I requested %" PRId64, i_actually_read, i_read );\
         free( p_buff ); \
         return( 0 ); \
     } \
